src/sections/Docker-Meshery/docker-extension-meshery.js

```diff
@@ -24,7 +24,7 @@ const DockerExtensionMeshery = () => {
           <h2 className="sectionTitle">
             Develop service mesh-ready apps using Docker Desktop and Meshery
           </h2>
-          <p>
+          <p className="sectionContent">
```

Comment:
This className should be applied to the `Row` component, as the header should also be padded. 
Though, a better fix for this is to set `margin: auto` in the CSS for the new class, that would fix the issue.

Reply:
Remove the className from here it is not required anymore.

Reply:
> Remove the className from here it is not required anymore.

Yes done :)